db/re/item_db.conf

```diff
@@ -135583,8 +135580,8 @@ item_db: (
 	Refine: false
 	ViewSprite: 844
 	Script: <"
-		bonus2 bSubRace,RC_Fish,15;
-		bonus3 bAddMonsterDropItem,579,RC_Fish,5;
+		bonus3(bSubDefEle, Ele_Water, 15, 3);
+		bonus3(bAddMonsterDropItem, Delicious_Fish, RC_Fish, 50);
```

Comment:
Just to make sure this isn't an error - is this change from 5 to 50 intentional?

Reply:
Ok, this also seems to be correct at 0.5% chance